src/app.js

```diff
@@ -19,15 +21,27 @@ const { getRemoteTabs, readError } = require(path.resolve(
 const pkg = require(path.resolve(__dirname, '../package.json'));
 const notifier = updateNotifier({ pkg });
 
+// Create a Configstore object. If conf file is empty, fill with default values instead and save to config path
+// Config path: ~/.config/configstore/check-it-out.json
+var conf = new Configstore(pkg.name, {});
+if (conf.size == 0) {
+  conf.set('logAlias', '--color=always');
+  conf.set('themeColor', '#FFA66');
+}
+
 if (notifier.update) {
   notifier.notify();
 }
 
 export const start = args => {
   if (args[0] === '-v' || args[0] === '--version') {
     process.stdout.write(pkg.version);
-
     process.exit(0);
+  } else if (args[0] === '-s' || args[0] === '--settings') {
```

Comment:
I think if we implement a way to change and update configs from within CIO, we may be adding in another level of complexity that should be split up to a separate ticket.

Reply:
Roger that, removing settings for now.